.vscode/settings.json

```diff
@@ -7,7 +7,7 @@
       }
     ],
     "editor.codeActionsOnSave": {
-      "source.fixAll.eslint": true
+      "source.fixAll.eslint": "explicit"
```

Comment:
any specific reason for using this?

Reply:
Its was changed by my vscode automatically. It can be changed back to true.